README.md

```diff
@@ -128,6 +128,66 @@ RUN zypper -n in gvim
 
 Examples taken from https://www.suse.com/documentation/sles-12/book_sles_docker/data/customizing_pre-build_images.html
 
+### Building images on non SLE distributions
+
+It is possible to build SLE based docker images on other distributions as well.
+For that the following two files from a base SLE system are needed:
+
+- `/etc/SUSEConnect` for the API access point
+- `/etc/zypp/credentials.d/SCCcredentials` for providing the user credentials
```

Comment:
You can add something after that like: "You can copy these files from a SLE machine that has been registered against SUSE Customer Center, RMT or SMT."